Library/ENV/4.3/cc

```diff
@@ -208,7 +207,7 @@ class Cmd
     return keep_orig?(path) unless ENV["HOMEBREW_EXPERIMENTAL_FILTER_FLAGS_ON_DEPS"]
 
     # Allow references to self
-    if keg_path && path.start_with?(keg_path)
+    if formula_prefix && path.start_with?(formula_prefix)
```

Comment:
I believe this should be `path.start_with?("#{formula_prefix}/")` if we wanted to be really strict, as we don't want `formula 12.3.1` to be able to access `Cellar/formula/12.3.11` or `Cellar/formula/12.3.1_2`, though that's probably never going to happen in practice.